wikimedia/wikipedia-ios
# Turn on watchlist feature flag in Experimental build

**Phabricator:** 
N/A

### Notes
This turns on the watchlist feature flag in our Experimental build, for design review. This also updates documentation with our new labels.

### Test Steps
1. Run app in simulator on Wikipedia, Staging, and Experimental schemes. Confirm you can see Watchlist under Account in Staging and Experimental schemes, and you cannot see it on Wikipedia scheme.
